### PR TITLE
Add demo for point sprites and custom attribs.

### PR DESCRIPTION
**Pull request by prideout**
Screen Shot 2019-07-10 at 11 22 57 AM

### CODE REVIEW

File: samples/point_sprites.cpp [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+#define TEXTURE_SIZE 128
+#define MAX_POINT_SIZE 128.0f
+
+void createSplatTexture(App& app, Engine* engine) {
+    static image::LinearImage splat(3, 3, 1);
+    splat.getPixelRef(1, 1)[0] = 1.0f;
+    splat = image::resampleImage(splat, TEXTURE_SIZE, TEXTURE_SIZE, image::Filter::GAUSSIAN_SCALARS);
+
+    Texture::PixelBufferDescriptor buffer(splat.getPixelRef(),
+            size_t(TEXTURE_SIZE * TEXTURE_SIZE * 4), Texture::Format::R, Texture::Type::FLOAT);
```
Comment (bejado) [inline samples/point_sprites.cpp]: Maybe use `sizeof(float)` here instead of 4 to help readability?

File: samples/point_sprites.cpp [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        kVertices[i].color = 0xff000000u | c | (c << 8u) | (c << 16u);
+        kPointSizes[i] = 12.0f + dsize * i;
+        kIndices[i] = i;
+    }
+
+    view->setClearColor({0.1, 0.125, 0.25, 1.0});
+    view->setPostProcessingEnabled(false);
+    view->setDepthPrepass(filament::View::DepthPrepass::DISABLED);
+
+    app.vb = VertexBuffer::Builder()
```
Comment (bejado) [inline samples/point_sprites.cpp]: Same `sizeof(float)` suggestion here with all the multiples of 4

File: samples/point_sprites.cpp (hunk 1 above)
Comment (prideout) [reply samples/point_sprites.cpp]: Thanks, fixed.

File: samples/point_sprites.cpp (hunk 2 above)
Comment (prideout) [reply samples/point_sprites.cpp]: Done.

**Pull request merged**